docs/1.0. Overview.md

```diff
@@ -32,7 +32,7 @@ The staged endpoint provides the means to make changes to settings associated wi
 
 #### Active
 
-The active endpoint reflects the current running state of the underlying Sender or Receiver. When a set of 'staged' settings is activated, these settings transition into the 'active' resource.
+The active endpoint reflects the current running configuration of the underlying Sender or Receiver. When a set of 'staged' settings is activated, these settings transition into the 'active' resource.
```

Comment:
There are a couple of instances of 'state' in 4.0. Behaviour (second para of [In-progress Activations](https://github.com/AMWA-TV/nmos-device-connection-management/blob/v1.1-dev/docs/4.0.%20Behaviour.md#in-progress-activations) and third para of [Scheduled Activations](https://github.com/AMWA-TV/nmos-device-connection-management/blob/v1.1-dev/docs/4.0.%20Behaviour.md#scheduled-activations)) which should also perhaps be 'configuration'?